tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -106,8 +126,19 @@ def setup_teardown(self, request):
         self.pv_obj = ocp.OCP(kind=constants.PV, namespace=self.namespace)
         self.nfs_sc = "ocs-storagecluster-ceph-nfs"
         self.sc = ocs.OCS(kind=constants.STORAGECLASS, metadata={"name": self.nfs_sc})
-        self.test_folder = "test_nfs"
-        utils.exec_cmd(cmd="mkdir -p " + self.test_folder)
+        self.run_id = config.RUN.get("run_id")
+        self.test_folder = f"mnt/test_nfs_{self.run_id}"
+        log.info(f"nfs mount point out of cluster is----- {self.test_folder}")
+        self.nfs_client_ip = config.ENV_DATA["nfs_client_ip"]
```

Comment:
If the nfs_client_ip will not be defined it will fail here with KeyError exception.

You have to use 
```suggestion
        self.nfs_client_ip = config.ENV_DATA.get("nfs_client_ip")
```

Reply:
Addressed in latest commit.